common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,126 @@
+"""OLI analytics service event tracker backend."""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from django.contrib.auth.models import User
+from student.models import anonymous_id_for_user
+from track.backends import BaseBackend
+from requests_oauthlib import OAuth1Session
+
+log = logging.getLogger(__name__)
```

Comment:
I think `pylint` treats this (`log`) as a constant, so we should type it in all-caps: `LOG`.